src/lib/mina_compile_config/mina_compile_config.mli

```diff
@@ -0,0 +1,49 @@
+val curve_size : int
```

Comment:
I felt it was better to explicitly give the `Mina_compile_config` interface because there are several values in here which are hard coded and seem like they should be considered as configuration